i18n/en.json

```diff
@@ -3231,7 +3231,6 @@
   "sidebar.createPublicPrivateChannel": "Create a public or private channel",
   "sidebar.directchannel.you": "{displayname} (you)",
   "sidebar.leave": "Leave the channel",
-  "sidebar.mainMenu": "Main Menu",
```

Comment:
I'm no sure it should be removed. If so, should appearances in other translations also be removed?

Reply:
Yes we should remove from other translation files if verified that its not used.

Reply:
I removed them.